js/interface.js

```diff
@@ -143,6 +144,25 @@ function convertTimeToMinutes () {
   return inputValue * selectValue;
 }
 
+// Shows warning if security setting are not configured correctly
+function checkSecurityRules () {
+  Fliplet.API.request('v1/apps/' + appId).then(function(result) {
+    if (!result || !result.app) {
+      return;
+    }
+    
+    var isSecurityConfigured = _.some(result.app.hooks, function(hook) {
```

Comment:
add a `var hooks = _.get(result.app, 'hooks', []);` just in case the app.hooks is undefined

Reply:
Fixed